### DISCUSSION TITLE
"Was this helpful" does not display on mobile or narrow width

### DISCUSSION THREAD

**Discussion by BobPritchett** [GitbookIO/community#610] [0. General bugs and questions] [answered]
The "Was this helpful?" prompt with three faces doesn't appear on the right side at narrow widths (1280 and less). On my 4k Windows display, set to 150% DPI scaling, it doesn't even appear when a Chrome window is half the width of the screen. (I have to make it slightly wider.) I would expect it to appear even with < 1280 width, and to appear below the article when the width gets to mobile sizes (where sidebar on left disappears).
![image](https://github.com/GitbookIO/community/assets/26860966/b41cd4af-57ab-4a94-9198-9a08fdc84693)
![image](https://github.com/GitbookIO/community/assets/26860966/a7ef2ca5-94de-4e63-9998-34db7265bc12)

**Comment by aikoii** [comment, marked as answer]
Hey @BobPritchett, thanks for flagging this! We already have a ticket open for this issue internally. We’ll keep you posted 👍
